apps/sms/views/conversation/js/attachment.js

```diff
@@ -26,8 +26,7 @@
   function Attachment(blob, options) {
     options = options || {};
     this.blob = blob;
-    this.name = blob.name || options.name ||
-      navigator.mozL10n.get('unnamed-attachment');
+    this.name = blob.name || options.name || 'undefined';
```

Comment:
I don't know. In all cases where Attachment is being used the name is there. The only place where there is no name is in tests where it just tests if it's a string.

So my thinking is that it should be '' or undefined and if you want to show something, we should do:
```
if (attachment.name) {
  element.textContent = attachment.name;
} else {
  element.setAttribute('data-l10n-id', 'unknown');
}
```
I just don't know where is the code that displays the name.